docs/dictionary/function/processor.lcdoc

```diff
@@ -23,17 +23,15 @@ if the processor is "ARM" then ...
 
 Returns (enum):
 
--   x86: Windows systems (any architecture) and Mac OS X and Linux
-    systems with a 32 bit processor.
--   x86_64: Mac and Linux systems running a 64 bit processor
--   i386: iOS Simulator.
--   ARM: iOS and Android Devices and the Android simulator
-
+-   x86: 32 or 64 bit Windows systems and 32 bit macOS or Linux systems.
```

Comment:
Interesting. So if it is built for **both** 32 and 64 bit architectures I would assume it to return whichever is being used by that app on that machine, x86 on a 32bit mc and x86_64 on a 64bit mc? When we ask for the `processor` in the IDE it will always respond `x86`because LC is only built for 32bit. I'll amend this entry to match its current functionality with a view to updating it if we change the call method to get the _actual_ processor type.